src/AppActions.js

```diff
@@ -1,7 +1,11 @@
 import * as ActionTypes from './AppConstants';
 import API from './Utilities/Api';
 
-const fetchData = async (url, headers, options) => (await API.get(url, headers, options)).data;
+const fetchData = async (url, headers, options) => {
+    await insights.chrome.auth.getUser();
```

Comment:
So most of the time, this would be a no-op right?

Reply:
yup thats, what Ive been told 🌮